### PR TITLE
Admin login

### PR DESCRIPTION
**Pull request by hallfox**
See #28. Also helps with #25.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,9 +1,17 @@
...
   - pip install -r requirements.txt
+before_script:
+  - npm install -g bower
+  - bower install
 script:
   - nosetests
   - pylint -E bikeshare_app
   - pylint -E tests
+env:
+  - BIKESHARE_APP_SECRET="\xf3& \x98\x9cy\xf6\xde6\x91\xd0\x8a\xd6\xbfF[\x86'6+J\x1cs\xfa"
```
Comment (jpalazz2) [inline .travis.yml:16]: Shouldn't we not have the secret key on github?

Comment (hallfox) [inline .travis.yml:16]: That is not the actual secret key, it's just one I generated for testing purposes. However, travis does allow us to use secure environment variables if we need to, so I suppose we could use the actual one, but as far as I see it's not necessary.

File: bikeshare_app/views.py [hunk 2]
```diff
@@ -1,26 +1,73 @@
...
+
+class User(flask_login.UserMixin):
+    """User model for login."""
+    pass
+
+
+@login_manager.user_loader
+def user_loader(email):
+    """Load a user from a session."""
+    # TODO: Hookup email verification to DB, verify they exist
```
Comment (jpalazz2) [inline bikeshare_app/views.py:18]: `User.objects(email=email).count() != 0`

Comment (jpalazz2) [inline bikeshare_app/views.py:18]: Although for now we should probably just leave the `TODO` since we have no way to add users....

**Comment by alanplotko** [comment]
@jpalazz2 @hallfox Did we create an issue for, or discuss, user registration?

We basically have two options:
- **Create our own simple registration process.**
  - Require email and password. Collect profile-like information (i.e. name, year, whatever BUBS needs). Complete server-side check for BU domain.
- **Tie in Google authentication.**
  - Require BU domain and pull some profile information from Google.

I'm all for tying in Google authentication, but we could also do the first one and not worry about integration. We don't need anything complex, as per the last time I've checked the BUBS spec sheet.

**Pull request merged**